def funcao():
    if condicao:
        raise Exception("Descri√ßao do erro")

try:
    funcao()
except Exception as e:
    print (f"Erro: {str(e)}")